cmd/gdeploy/commands/status.go

```diff
@@ -17,18 +17,21 @@ var StatusCommand = cli.Command{
 		if err != nil {
 			return err
 		}
-		o, err := dc.LoadOutput(ctx)
+
+		ss, err := dc.GetStackStatus(ctx)
 
 		if err != nil {
 			return err
 		}
-		o.PrintTable()
 
-		ss, err := dc.GetStackStatus(ctx)
+		//wait until we know there are no errors then print the status
+
+		o, err := dc.LoadOutput(ctx)
```

Comment:
@meyerjrr in this case I think we always want it to print the table event if the status is bad.
Otherwise, the user can never access the outputs without going into the AWS console.

Reply:
I missed this in my review but yep, the status command should always print a table regardless of the status. Maybe we can print the status after we print the table. The specific user feedback we've got is that the create/update commands print a table, which obscures the actual deployment status.